<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumsWindow</class>
 <widget class="QMainWindow" name="AlbumsWindow">
  <property name="windowTitle">
   <string>–ê–ª—å–±–æ–º—ã</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <!-- üîç –§–∏–ª—å—Ç—Ä –∏ –ø–æ–∏—Å–∫ -->
    <item>
     <widget class="QWidget" name="filterWidget">
      <layout class="QHBoxLayout" name="filterLayout">
       <item>
        <widget class="QLabel" name="labelSearch">
         <property name="text">
          <string>–ü–æ–∏—Å–∫:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSearch"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFilter">
         <item><property name="text"><string>–í—Å–µ –ø–æ–ª—è</string></property></item>
         <item><property name="text"><string>–ù–∞–∑–≤–∞–Ω–∏–µ</string></property></item>
         <item><property name="text"><string>–ò—Å–ø–æ–ª–Ω–∏—Ç–µ–ª—å</string></property></item>
         <item><property name="text"><string>–ñ–∞–Ω—Ä</string></property></item>
         <item><property name="text"><string>–ì–æ–¥ –≤—ã–ø—É—Å–∫–∞</string></property></item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- üìã –¢–∞–±–ª–∏—Ü–∞ -->
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>

    <!-- üîò –ö–Ω–æ–ø–∫–∏ -->
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>–°–æ–∑–¥–∞—Ç—å –∞–ª—å–±–æ–º</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>–†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>–£–¥–∞–ª–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenAlbum">
        <property name="text">
         <string>–û—Ç–∫—Ä—ã—Ç—å –∞–ª—å–±–æ–º</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
